data/json/monstergroups.json

```diff
@@ -9,7 +9,14 @@
     "type":"monstergroup",
     "name" : "GROUP_FOREST",
     "default" : "mon_null",
-    "monsters" : [
+    "monsters" : [ { "monster" : "mon_wild_beehive", "freq" : 1, "cost_multiplier" : 10, "pack_size" : [1,1], "conditions" : ["DAY","DUSK","NIGHT","DAWN","AUTUMN", "WINTER"] },
+      { "monster" : "mon_wild_beehive_empty", "freq" : 1, "cost_multiplier" : 20, "pack_size" : [1,1], "conditions" : ["DAY","DUSK","NIGHT","DAWN","SPRING"] },
```

Comment:
The changes here should probably go into the mod folder. Some of the monster don't exists outside of the mod, but this file is part of the core data.